sBTC SIP

```diff
@@ -0,0 +1,166 @@
+| sBTC holder              | A person or system holding sBTC, interested in turning it back into BTC. |
+| .sbtc contract           | A smart contract (or a collection of contracts) defining the sBTC token and functions related to it. |
+| sBTC UTxO                | The single UTxO holding the entire BTC balance that’s pegged into sBTC. This UTxO is managed and maintained by the sBTC Signers. |
+| Stacks Signer            | The standard Stacks Signer introduced in Nakamoto. |
+| sBTC Bootstrap Signer    | A completely separate signer from the Stacks signer that will sign sBTC operations and communicate with contracts on the chain to make that feasible. This entity is registered as an sBTC signer and has partial access to spending the sBTC UTxO. This only exists in sBTC-v1. In subsequent sBTC releases this functionality will be native in the Stacks Signer. |
+| sBTC Signer              | sBTC Signer is a more general term for the entity that signs sBTC transactions under any version of sBTC. Under sBTC-v1 the sBTC signer is the sBTC Bootstrap Signer which is wholly separate from the Stacks Signer, but in subsequent releases of sBTC the Stacks Signer will also be responsible for signing sBTC operations, so it would be the sBTC Signer. |
+| sBTC Bootstrap Signer Set| The set of all sBTC signers. Each is registered with the .sbtc contract and the transfer. These entities as a group have full democratic access to the sBTC UTxO. |
+| sBTC Signer API          | API exposed by the sBTC Signer binary that handles basic low level commands. Remains private except to the Deposit API. |
+| Deposit API              | A third party API that communicates with the sBTC Bootstrap Signers via the sBTC Signer API. |
+
+## Problem Statement
+Bitcoin is limited in its programmability and scalability. While its security and censorship-resistance make it an ideal platform to build decentralized applications, it is not able to scale to meet the modern needs of users and developers. These challenges have been largely driven by the ethos of the Bitcoin ecosystem to prioritize simplicity and decentralization at the base layer.
+
+**Limitations with Bitcoin’s Scripting Language:** Bitcoin’s script has limited programmability, making it unsuitable for most decentralized applications. This leaves users with very few options to use financial applications, like lending and trading, without entrusting their Bitcoin to centralized entities. This also exposes users to counterparty risk, which has the potential to result in lost funds. This centralization contradicts the fundamental principles of Bitcoin and dampens the potential to unlock new functionality and scalability for users. These limitations in Bitcoin's script have made it challenging for developers to build applications directly on Bitcoin and led developers to seek alternative ecosystems that provide them with greater flexibility and tooling.
```

Comment:
"unsuitable" sounds subjective.  You'd be surprised at the things you _can_ do with BTC script.  The "aha" moment for why you'd need sBTC is that some applications need things that only smart contracts can provide -- namely, public global persistent state.  I think you should come up with a concrete example of an application that _cannot_ be implemented with BTC script, thereby _necessitating_ sBTC.  The way to do this is to show that the example application requires all of the trappings of the Stacks blockchain in order to be implemented.

Reply:
"This centralization contradicts the fundamental principles of Bitcoin"

This is out of scope.  Who are we to tell others what the "fundamental principles" of Bitcoin are, and why "centralization" "contradicts" them?  Bitcoin provides different value to different people; Bitcoin itself is an inanimate software artifact that cannot have principles.